src/test/regress/bin/normalize.sed

```diff
@@ -161,6 +161,19 @@ s/Citus.*currently supports/Citus currently supports/g
 s/prepared transaction with identifier .* does not exist/prepared transaction with identifier "citus_x_yyyyyy_zzz_w" does not exist/g
 s/failed to roll back prepared transaction '.*'/failed to roll back prepared transaction 'citus_x_yyyyyy_zzz_w'/g
 
+# Table aliases for partitioned tables in explain outputs might change
```

Comment:
can we remove these rules when we drop pg 11 support? So, maybe add the following so that we can remove  that rule while dropping the  test output?

# can be removed when we remove PG_VERSION_NUM >= 110000

Reply:
It is both related with pg11&pg12.
So I can add a comment to mention those both versions